Classes/Domain/Model/Product.php

```diff
@@ -923,6 +969,28 @@ public function _getAllAttributesSets(): array // @codingStandardsIgnoreEnd
         );
     }
 
+    /**
+     * Return all attributes sets from choosen productType.
+     * It fetch every attribute set of every category from parents tree
+     * + product own attributes sets.
+     *
+     * @return AttributeValues[]
+     * @internal Use internally to map all attribute values for frontend rendering.
+     * @see getAttributesValues
+     */
+
+    /** @codingStandardsIgnoreStart */
+    public function _getAllAttributeValues(): array // @codingStandardsIgnoreEnd
+    {
+        if (empty($this->attributesValues)) {
```

Comment:
The variable `$this->attributesValues` should really be called `$this->attributeValues` without the `s` in `attributes`.